apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/config/PortalConfig.java

```diff
@@ -273,4 +275,12 @@ public String getAdminServiceAccessTokens() {
   public boolean supportSearchByItem() {
     return getBooleanProperty("searchByItem.switch", true);
   }
+  
+  public List<String> listOFCodeFragment() {
+    String[] value = getArrayProperty("auth.userPasswordChecker.listOFCodeFragment", null);
```

Comment:
It may be better to change OF to Of?
BTW, there wasn't a good standard naming pattern for configurations. However, recently we thought it may look better if we separate the word with `-`, what do you think?
```suggestion
  public List<String> listOfCodeFragment() {
    String[] value = getArrayProperty("auth.user-password-checker.list-of-code-fragment", null);
```

Reply:
Done.